java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -444,6 +448,32 @@ public static HostColumnVector timestampNanoSecondsFromLongs(long... values) {
     return build(DType.TIMESTAMP_NANOSECONDS, values.length, (b) -> b.appendArray(values));
   }
 
+  /**
+   * Create a new decimal vector from unscaled values (int array) and scale.
+   * The created vector is of type DType.DECIMAL32, whose max precision is 9.
+   * Compared with scale of [[java.math.BigDecimal]], the scale here represents the opposite meaning.
+   */
+  public static HostColumnVector decimalFromInts(int scale, int... values) {
+    if (-scale > DType.DECIMAL32_MAX_PRECISION) {
```

Comment:
So I am a bit confused but how exactly does scale impact precision?

Lets say that we have the following

unscaled value = 1
scale = -20

result should be the same as `unscaled_value * 10 ^ scale` or  0.00000000000000000001

The precision is just about how many digits the unscaled value can hold.  The only limit on scale we need to worry about is if it fits in an `int32_t`, which this interface by definition enforces.

Reply:
Yes, the scale check here is unnecessary. Basically. Precision is independent from the scale. I've removed these checks. Sorry for making confusion.